conf/modules.config

```diff
@@ -544,6 +544,47 @@ process {
             mode: params.publish_dir_mode,
         ]
     }
+
+}
+
+//
+// Score and rank structural variants
+//
+
+process {
+    withName: '.*RANK_VARIANTS_SV:GENMOD_ANNOTATE' {
+        ext.args = " --annotate_regions "
+        publishDir = [
+            path: { "${params.outdir}/genmod" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
```

Comment:
Not sure we need to publish all the intermediary files. I think the main file of interest is the resulting vcf file from the "compound" module.

Reply:
Will fix this 👍🏻